integration_tests/integration/instructions-confirm.spec.ts

```diff
@@ -77,8 +77,12 @@ context('Instructions Confirmation', () => {
     instructionsPage.subHeading().should('contain', 'Review allocation instructions')
     instructionsPage
       .label()
-      .should('contain', 'These notes will automatically be sent to John Doe (john.doe@test.justice.gov.uk)')
+      .should(
+        'contain',
```

Comment:
We really ought to have a test to verify what we send to the server.

Reply:
I appreciate it is [hard to detect](https://github.com/ministryofjustice/manage-a-workforce-ui/blob/2a88df39c22afa3acf22221868bb00f537f26082/integration_tests/mockApis/allocationComplete.ts#L37) in amongst the monstrously large PR, but there is one of sorts. It's not actually an assertion - it's a wiremock matcher - but if you do not check the checkbox (or vice versa), the test does fail:

Reply:
see also [this check for the opposite condition](https://github.com/ministryofjustice/manage-a-workforce-ui/blob/2a88df39c22afa3acf22221868bb00f537f26082/integration_tests/mockApis/allocationComplete.ts#L13)